packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -37,9 +37,20 @@ const AccountSwitcher = props => {
     const [is_deriv_demo_visible, setDerivDemoVisible] = React.useState(true);
     const [is_deriv_real_visible, setDerivRealVisible] = React.useState(true);
     const [is_dmt5_demo_visible, setDmt5DemoVisible] = React.useState(true);
-    const [is_dmt5_real_visible, setDmt5RealVisible] = React.useState(true);
+    const [is_dmt5_real_visible, setDmt5RealVisible] = React.useState(false);
 
     const wrapper_ref = React.useRef();
+    const mtd5_ref = React.useRef(null);
```

Comment:
Should this be called `dmt5_ref` instead?

Reply:
changed pls accept:)